### ISSUE TITLE
German number.percentage.format.precision causes error because it is a String, not a Fixnum

### ISSUE THREAD

**Issue by florianguenther** [solidusio-contrib/solidus_i18n#60]
in file `config/locales/de.yml`

```
number:
  percentage:
    format:
      precision: '1'
```

should be changed to

```
number:
  percentage:
    format:
      precision: 1
```

**Comment by jugglinghobo** [comment]
I ran into this as well.
The 'offending' code in `Spree::TaxRate#amount_for_adjustment_label`:

```ruby
def amount_for_adjustment_label
  ActiveSupport::NumberHelper::NumberToPercentageConverter.convert(
    amount * 100,
    locale: I18n.locale # in this case :de
  )
end
```

Relevant Rails code:
* In `NumberHelper`, precision value is [read from the locale file](https://github.com/rails/rails/blob/4-2-stable/activesupport/lib/active_support/number_helper/number_converter.rb#L157) and set as `:presicion` option.

* in rails master, `precision` is [converted in case it's not a number](https://github.com/rails/rails/blob/master/activesupport/lib/active_support/number_helper/number_to_rounded_converter.rb#L10-L18), but at least in rails [4.2-stable](https://github.com/rails/rails/blob/4-2-stable/activesupport/lib/active_support/number_helper/number_to_rounded_converter.rb#L8), no such conversion happens.

I haven't found any open PRs containing a fix for this, would you accept one that contains this fix and a spec for it, or do you need more info as stated in your contribution guidelines?

**Comment by felixbuenemann** [comment]
See PR #67.

**Comment by fschwahn** [comment]
This is important because it breaks the checkout process with german locale.

Also @jugglinghobo 
> in rails master, precision is converted in case it's not a number, but at least in rails 4.2-stable, no such conversion happens.

This is not correct, if you take a look at the code you linked it just convert the number itself, not the precision argument. That means this is not going away with a rails update and needs to be fixed.

**Comment by fschwahn** [comment]
This can be closed, as #67 is merged 👍

**Comment by tvdeyen** [comment]
Thanks for the reminder @fschwahn
